pypsa/networkclustering.py

```diff
@@ -239,6 +254,10 @@ def get_clustering_from_busmap(network, busmap, with_time=True, line_length_fact
     io.import_components_from_dataframe(network_c, buses, "Bus")
     io.import_components_from_dataframe(network_c, lines, "Line")
 
+    # Carry forward global constraints to clustered network.
+    if global_constraints:
+        network_c.global_constraints = network.global_constraints
+    
```

Comment:
```suggestion
    # Carry forward global constraints to clustered network.
    network_c.global_constraints = network.global_constraints
    
```
If think it make more sense to carry along global constraint in any cases.